doc/doxygen_project.txt.in

```diff
@@ -72,7 +72,7 @@ WARN_LOGFILE           =
 #---------------------------------------------------------------------------
 # configuration options related to the input files
 #---------------------------------------------------------------------------
-INPUT                  = $MRPT_LIST_INPUT
+INPUT                  = $MRPT_LIST_INPUT "../libs/vision/src/pnp"

```

Comment:
@mangi020 Have you tried to run `build_docs.sh -h` with the original line (without your added path)?? I think it should work, so the simper the better... right??